server/docs/source/index.html.md

```diff
@@ -286,15 +286,15 @@ follow | Stream logs
 
 ```json
 {
-	"id": "mygrid/misty-sun-87",
-	"node_id": "RQKP:Y32W:SB4H:7TNG:5BKC:R6ZO:5B25:C2AV:3Z3Q:SVPX:A76C:WPBX",
+	"id": "RQKP:Y32W:SB4H:7TNG:5BKC:R6ZO:5B25:C2AV:3Z3Q:SVPX:A76C:WPBX",
```

Comment:
This is really an API bug, it should be like it is in the documentation, but unfortunately it isn't. 

The `/v1/nodes/RQKP:Y32W:SB4H:...` API exists, but it it's deprecated and odd: it only supports GET/PUT, and it requires the `Kontena-Grid-Token` header.